### PR TITLE
Ahayel/fill passed tests 51

### PR DESCRIPTION
**Pull request by artemgael96**

### CODE REVIEW

**Review by ElenaBurak — APPROVED** [review]

File: exalearn-api/ExaLearn.Dal/Database/ExaLearnDbContext.cs [hunk 1]
```diff
@@ -28,7 +30,140 @@ public ExaLearnDbContext(DbContextOptions<ExaLearnDbContext> options) : base(opt
         protected override void OnModelCreating(ModelBuilder builder)
         {
             base.OnModelCreating(builder);
+
+            var password = "_Test1234";
```
Comment (romanstat) [inline exalearn-api/ExaLearn.Dal/Database/ExaLearnDbContext.cs]: put into method extension

File: exalearn-api/ExaLearn.Dal/Entities/Role.cs [hunk 2]
```diff
@@ -0,0 +1,8 @@
+﻿using Microsoft.AspNetCore.Identity;
+
+namespace ExaLearn.Dal.Entities
+{
+    public class Role : IdentityRole<int>
```
Comment (romanstat) [inline exalearn-api/ExaLearn.Dal/Entities/Role.cs:5]: ?

File: exalearn-api/ExaLearn.Dal/Entities/UserRole.cs [hunk 3]
```diff
@@ -0,0 +1,8 @@
+﻿using Microsoft.AspNetCore.Identity;
+
+namespace ExaLearn.Dal.Entities
+{
+    public class UserRole : IdentityUserRole<int>
```
Comment (romanstat) [inline exalearn-api/ExaLearn.Dal/Entities/UserRole.cs:5]: ?

File: exalearn-api/ExaLearn.Dal/Database/ExaLearnDbContext.cs [hunk 4]
```diff
@@ -28,7 +30,140 @@ public ExaLearnDbContext(DbContextOptions<ExaLearnDbContext> options) : base(opt
...
+                UserId = user.Id
+            };
+
+            var identityHrRole = new UserRole
+            {
+                RoleId = hrRole.Id,
+                UserId = hrUser.Id
+            };
+
+            var identityCoachRole = new UserRole
```
Comment (romanstat) [inline exalearn-api/ExaLearn.Dal/Database/ExaLearnDbContext.cs]: new IdentityUserRole

File: exalearn-api/ExaLearn.Dal/Database/ExaLearnDbContext.cs [hunk 5]
```diff
@@ -28,7 +30,140 @@ public ExaLearnDbContext(DbContextOptions<ExaLearnDbContext> options) : base(opt
...
+
+            var hrRole = new Role
+            {
+                Id = 2,
+                Name = RoleNames.Hr,
+                NormalizedName = RoleNames.Hr.ToUpper(),
+                ConcurrencyStamp = Guid.NewGuid().ToString()
+            };
+
+            var coachRole = new Role
```
Comment (romanstat) [inline exalearn-api/ExaLearn.Dal/Database/ExaLearnDbContext.cs]: new IdentityRole

File: exalearn-api/ExaLearn.Dal/Database/ExaLearnDbContext.cs (hunk 1 above)
Comment (artemgael96) [reply exalearn-api/ExaLearn.Dal/Database/ExaLearnDbContext.cs]: have done

**Review by romanstat — APPROVED** [review]

**Pull request merged**